pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -403,9 +403,20 @@ func (c *Cluster) startDeployments(mons []*monConfig, requireAllInQuorum bool) e
 		if err != nil {
 			return fmt.Errorf("failed to create mon %s. %+v", mons[i].DaemonName, err)
 		}
+		// For the initial deployment (first creation) it's expected to not have all the monitors in quorum
+		// However, in an event of an update, it's crucial to proceed monitors by monitors
+		// At the end of the method we perform one last check where all the monitors must be in quorum
```

Comment:
This comment make sense, but it is hard to place in context. It seems to imply that there is a change in behavior in this method when an initial deployment is made (not all monitors are in quorum is ok), but I'm not seeing any conditional behavior.

Reply:
The condition here is managed by turning on and off `requireAllInQuorum`.